src/cursor.ts

```diff
@@ -113,77 +139,87 @@ var Cursor = P(Point, function(_) {
     self.jQ.addClass('mq-cursor');
     return offset;
   }
-  _.unwrapGramp = function() {
+  unwrapGramp () {
     var gramp = this.parent.parent;
     var greatgramp = gramp.parent;
     var rightward = gramp[R];
     var cursor = this;
 
     var leftward = gramp[L];
     gramp.disown().eachChild(function(uncle) {
-      if (uncle.isEmpty()) return;
+      if (uncle.isEmpty()) return true;
```

Comment:
It doesn't look like this return value is used by the one caller I see, should this be declared a void method?

Reply:
Possibly. I think this would be fair game to cleanup assuming things didn't change significantly in the PR that removes jquery.